src/services/OswServiceTaskWiFi.cpp

```diff
@@ -115,6 +151,22 @@ void OswServiceTaskWiFi::loop() {
     }
 }
 
+void OswServiceTaskWiFi::selectCredentials() {
+    if(this->m_connectFailureCount % 4 == 0 and !OswConfigAllKeys::wifiSsid.get().isEmpty()) {
+        this->m_clientSSID = OswConfigAllKeys::wifiSsid.get();
+        this->m_clientPass = OswConfigAllKeys::wifiPass.get();
+    } else if(this->m_connectFailureCount % 4 == 1 and !OswConfigAllKeys::fallbackWifiSsid1st.get().isEmpty()) {
+        this->m_clientSSID = OswConfigAllKeys::fallbackWifiSsid1st.get();
+        this->m_clientPass = OswConfigAllKeys::fallbackWifiPass1st.get();
+    } else if(this->m_connectFailureCount % 4 == 2 and !OswConfigAllKeys::fallbackWifiSsid2nd.get().isEmpty()) {
+        this->m_clientSSID = OswConfigAllKeys::fallbackWifiSsid2nd.get();
+        this->m_clientPass = OswConfigAllKeys::fallbackWifiPass2nd.get();
+    } else if(this->m_connectFailureCount % 4 == 3) {
```

Comment:
Just as a sanity check - or in case someone would like to add something here. I also think it feels more complete and shows better that we only have x amount of options...